assets/js/modules/reader-revenue-manager/components/common/ProductIDSelect.test.js

```diff
@@ -40,6 +45,8 @@ describe( 'ProductIDSelect', () => {
 				productID: 'product-1',
 				productIDs,
 			} );
+
+		muteFetch( publicationsEndpoint );
```

Comment:
We generally prefer our tests not to need to know about the underlying endpoint requests, unless necessary, so it would be preferable to fix this test like so. I'll apply this myself to save the round trip.

```suggestion
		registry
			.dispatch( MODULES_READER_REVENUE_MANAGER )
			.receiveGetPublications( publications );
```